hack/test.sh

```diff
@@ -10,104 +10,97 @@ set -eo nounset
 shopt -s nullglob
 
 IMAGE=${IMAGE:-openshift/openldap-candidate}
+RUNTIME=${RUNTIME:-podman}
+PORT=
 
 CIDFILE_DIR=$(mktemp --suffix=openldap_test_cidfiles -d)
 
 function cleanup() {
   for cidfile in $CIDFILE_DIR/* ; do
     CONTAINER=$(cat $cidfile)
     echo "Stopping and removing container $CONTAINER..."
-    docker stop $CONTAINER
-    exit_status=$(docker inspect -f '{{.State.ExitCode}}' $CONTAINER)
 
+    $RUNTIME stop $CONTAINER
+    exit_status=$($RUNTIME inspect -f '{{.State.ExitCode}}' $CONTAINER)
     if [ "$exit_status" != "0" ]; then
       echo "Dumping logs for $CONTAINER"
-      docker logs $CONTAINER
+      $RUNTIME logs $CONTAINER
     fi
-
-    docker rm $CONTAINER
+    $RUNTIME rm $CONTAINER
     rm $cidfile
-
     echo "Done."
   done
+
   rmdir $CIDFILE_DIR
 }
 
 function get_cid() {
   local id="$1" ; shift || return 1
-  echo $(cat "$CIDFILE_DIR/$id")
-}
 
-function get_container_ip() {
-  local id="$1"
-  docker inspect --format='{{.NetworkSettings.IPAddress}}' $(get_cid "$id")
+  echo $(cat "$CIDFILE_DIR/$id")
 }
 
 function test_connection() {
   local name=$1
+  local port=$2
+  echo "  Testing OpenLDAP connection to localhost:$port..."
+
   local max_attempts=20
   local sleep_time=2
-
-  ip=$(get_container_ip $name)
```

Comment:
wasn't the original better in case we're not running within hostnetwork?

Reply:
There is no exposed IP with [podman](https://podman.io/getting-started/network#rootless-networking). It was part of the debugging of `$(RUNTIME)` change :smile: